tfbpmodeling/lasso_modeling.py

```diff
@@ -397,7 +402,8 @@ def get_modeling_data(
         :param drop_intercept: If `drop_intercept` is True, "-1" will be appended to
             the formula string. This will drop the intercept (constant) term from
             the model matrix output by patsy.dmatrix. Default is `False`.
```

Comment:
```suggestion
            the model matrix output by patsy.dmatrix. Default is `False`. Note
            that if this is `False`, but `center_scale` is `True`, then the
            StandardScaler `with_mean = False` and the data is only scaled,
            not centered.
```

This was a subtle use of `drop_intercept` -- always comment, in the code, and document these sort of subtleties